### PR TITLE
Add parameter to print_html to log to sessions log

### PR DESCRIPTION
**Pull request by libraryaddict**
My main concern about this PR is that I can't actually test multiline support properly.

The way the runtime functions are executed currently results in a malformed command if we attempt to add a newline.

Try run for example, `js cliExecute("ash print('Test\nMessage');")`

Session stream testing can also be cleaner if we expose the stream setter.

### CODE REVIEW

File: test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java [hunk 1]
```diff
@@ -123,6 +128,51 @@ void floundryLocations() {
...
+                        <td>
+                          <p>
+                            roninStoragePulls
+                          </p>
+                        </td>
+                      </tr>
+                    </table>"""
+              .replaceAll("\r|\n", "<br>"); // The command tests don't handle newlines very well
+
+      // Confirm that print_html doesn't log to session
```
Comment (gausie) [inline test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java:161]: Split these tests

File: src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java [hunk 2]
```diff
@@ -2959,6 +2966,18 @@ public static Value print(ScriptRuntime controller, final Value string, final Va
     return DataTypes.VOID_VALUE;
   }
 
+  public static Value print_html(
+      ScriptRuntime controller, final Value string, final Value logToSession) {
+    if (logToSession.intValue() == 1) {
+      String parameters = StringUtilities.stripHtml(string.toString());
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:2972]: Parameters makes less sense as a variable name here

**Review by gausie — COMMENTED** [review]
Looks good to me, pending these small changes

**Comment by libraryaddict** [comment]
Actually, that's a thought.

I'm stripping away newlines as done in print. But I'm not sure that makes as much sense here.
One alternative, for example. Would be to split each line and print them individually.

**Review by jaadams5 — APPROVED** [review]

**Comment by jaadams5** [comment]
I made a bunch of local modifications to the mirror tests and they passed or failed as I expected.  No real benefit to keeping them but that addressed my concerns.

**Pull request merged**